protos/payload.proto

```diff
@@ -15,9 +15,12 @@ message ClaimPayload {
     EDIT_CLAIM = 3;
   }
 
+  // JSON serialized UTC timestamp.
+  string timestamp = 1;
```

Comment:
We'll use this in the frontend to visually show (in EST) when the claim was initially created.